requirements.txt

```diff
@@ -0,0 +1,4 @@
+Flask==2.0.2
+requests==2.27.1
+pytest==6.2.5
```

Comment:
There isn't a pytest in the readme originally as it was something that would be assessed in the test (tdd etc), assume it was left out originally as can use unit test instead etc (50/50 split in the community over pytest and unit test atm)

Reply:
Stand corrected was added after I joined. I mean if we want it super vanilla then it's all about the unit tests